services/galley/src/Galley/API/Update.hs

```diff
@@ -587,13 +587,13 @@ rmBot (zusr ::: zcon ::: req) = do
 -------------------------------------------------------------------------------
 -- Helpers
 
-addToConversation :: ([BotMember], [Member]) -> UserId -> ConnId -> [UserId] -> Data.Conversation -> RoleName -> Galley Response
-addToConversation _              _   _    [] _ _    = return $ empty & setStatus status204
-addToConversation (bots, others) usr conn xs c role = do
+addToConversation :: ([BotMember], [Member]) -> (UserId, RoleName) -> ConnId -> ([UserId], RoleName) -> Data.Conversation -> Galley Response
```

Comment:
Shouldn't this be:

```suggestion
addToConversation :: ([BotMember], [Member]) -> (UserId, RoleName) -> ConnId -> ([UserId, RoleName]) -> Data.Conversation -> Galley Response
```
? There's no requirement to add all users to a conversation with the same conversation role, or is there?

Reply:
Yes, this actually makes the code a lot cleaner https://github.com/wireapp/wire-server/pull/927/commits/1b4768d763f52092ec97981c2787f9a62828ce1d

To be precise, it allows me to fully remove the section:
```
    userRole u	
       | u == orig = origRole	
       | otherwise = othersRole
```
and have the users to be added to the conversation fully qualified, thanks 👍